prisma/schema.prisma

```diff
@@ -72,6 +81,11 @@ model Product {
     images   Image[]          @relation("ProductImages")
     variants ProductVariant[]
 
+    createdBy   User   @relation(fields: [createdById], references: [id])
+    createdById String
```

Comment:
Why do we need a this for a product?

Reply:
It was just an additional parameter and will be removed